apps/indexer/src/handlers/Strategy.ts

```
@@ -11,6 +11,7 @@ Strategy.TimestampsUpdatedWithRegistrationAndAllocation.handler(async ({}) => {}
 
 // DistributionUpdated Handler
 Strategy.DistributionUpdated.handler(async ({}) => {});
+Strategy.DistributionUpdatedWithMerkleRoot.handler(async ({}) => {});
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)